app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,47 @@
+class AnimalsController < ApplicationController
+
+
+def index
+  animals = Animal.all
+  render json: animals
+end
```

Comment:
Good job on the 2nd story, you will properly be able to list all of the animals once created with Animal.all render json: animals